modules/cache_behavior/README.md

```diff
@@ -0,0 +1,5 @@
+# Cache Behaviors submodule for S3 Website
+
+This submodule is intended to be a convenient way to create well validated cache behaviors for the S3 website module.
+
+See the main module's readme for example usage.
```

Comment:
A readme is required for this submodule to be treated as a public submodule by the terraform registry.